src/applications/static-pages/social-share-links.js

```diff
@@ -0,0 +1,41 @@
+/*
+Open share links in a new modal window
+*/
+
+export function openShareLink() {
+  const shareLinks = document.querySelectorAll('.va-js-share-link');
+  const hasNavigatorShare = navigator.share !== undefined;
+  const metaTitle = document.querySelector("meta[name='title']").content;
+  const metaDescription = document.querySelector("meta[name='description']")
+    .content;
+  const metaUrl = document.querySelector("meta[property='og:url']").content;
+
+  shareLinks.forEach(link => {
+    link.addEventListener('click', event => {
+      if (!hasNavigatorShare) {
+        // Desktop share
+        window.open(
+          link.href,
+          'social_popup',
+          'left=20,top=20,width=700,height=500,toolbar=0,menubar=0,status=0,location=0,resizable=0',
+        );
+      } else {
+        // Android native share
+        navigator
+          .share({
+            title: metaTitle,
+            text: metaDescription,
+            url: metaUrl,
+          })
+          .then(() => event.preventDefault())
```

Comment:
Would putting `event.preventDefault()` at the top of the function and removing it from here and the end of the function work? It looks like you always want to prevent the default behavior, but this will do it twice sometimes.